app/views/vacancies/campaign_landing_page.html.slim

```diff
@@ -0,0 +1,63 @@
+- content_for :skip_links do
+  = govuk_skip_link(href: "#search-results", text: t("jobs.skip_link_list"))
+
+- content_for :breadcrumbs do
+  = govuk_breadcrumbs breadcrumbs: organisation_landing_page_breadcrumbs(@vacancies_search.organisation_slug) if @vacancies_search.organisation_slug.present?
+
+= render "vacancies/search/page_title_and_description", landing_page: @landing_page
+
+h1 class="govuk-heading-l" role="heading" aria-level="1"
+  = t("jobs.search_result_heading", count: number_with_delimiter(@vacancies_search.total_count))
+
+- any_vacancies = @vacancies_search.total_count.positive?
+- if @vacancies_search.active_criteria? && any_vacancies
+  p = t("subscriptions.link.help_text_html", link: govuk_link_to(t("subscriptions.link.text"), new_subscription_path(search_criteria: @vacancies_search.active_criteria, coordinates_present: @vacancies_search.point_coordinates.present?)))
+
+= form_for @form, as: "", url: jobs_path, method: :get, html: { data: { controller: "form" }, role: "search" } do |f|
+  .custom-banner-background
+    .govuk-width-container
+      .custom-banner
+        .govuk-grid-row
+          .govuk-grid-column-one-third-at-desktop
+            h1.govuk-heading-l class="govuk-!-margin-bottom-4" = "#{@jobseeker_name}, find your primary teacher job"
```

Comment:
It looks like we've only implemented the 'primary teacher' landing page here. Do we not need to parameterise this so we can do secondary teachers (and others)?

Reply:
yes, could we make it so it pulls through either 'primary' or 'secondary', depending on their info in Mailchimp?